homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -150,6 +150,12 @@ def convert_and_round(
         unit_type=PERCENTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "climate_status": BMWSensorEntityDescription(
+        key="activity",
+        translation_key="climate_status",
+        key_class="climate",
+        icon="mdi:fan",
+    ),
```

Comment:
This requires #110294 and an availability check on `vehicle.is_remote_climate_stop_enabled`.

Reply:
@rikroe added now, not sure how I can define possible states here, any hint, please? Thanks

Reply:
`options=[...]`